test/e2e/config/e2e_conf.yaml

```diff
@@ -73,6 +73,7 @@ providers:
 variables:
   KUBERNETES_VERSION: "${KUBERNETES_VERSION}"
   CNI: "./data/cni/calico/calico.yaml"
+  IRONIC_HOST_IP: "172.22.0.2"
```

Comment:
IMO we have this var used in CI in m3-dev-env, can we use that by calling directly wherever instead  of hard-coding here?
`CLUSTER_PROVISIONING_IP` or `CLUSTER_URL_HOST`?